templates/contrib/facets/facets-item-list--range-slider--leisure-age.html.twig

```diff
@@ -0,0 +1,5 @@
+{% extends 'facets-item-list--range-slider.html.twig' %}
+
+{% block pipscontent %}
+  {{ 'From %min to %max year'|t({'%min': '', '%max': ''}) }}
+{% endblock pipscontent %}
```

Comment:
Deze stad.gent specifieke templates horen niet thuis in Gent Base omdat andere projecten hier niets aan hebben. Dit is niet generiek genoeg. Daarom zou ik ze in sg_theme zetten (jaar en euro slider).